packages/broker/test/integration/plugins/operator/MaintainOperatorValueService.test.ts

```diff
@@ -1,125 +1,108 @@
 import { MaintainOperatorValueService } from '../../../../src/plugins/operator/MaintainOperatorValueService'
-import { OperatorServiceConfig } from '../../../../src/plugins/operator/OperatorPlugin'
-import { getProvider } from './smartContractUtils'
-import { createClient, createTestStream } from '../../../utils'
-import { setupOperatorContract } from './setupOperatorContract'
 
 const chainConfig = CHAIN_CONFIG.dev2
-const theGraphUrl = `http://${process.env.STREAMR_DOCKER_DEV_HOST ?? '127.0.0.1'}:8800/subgraphs/name/streamr-dev/network-subgraphs`
 
+const theGraphUrl = `http://${STREAMR_DOCKER_DEV_HOST}:8800/subgraphs/name/streamr-dev/network-subgraphs`
 const logger = new Logger(module)
 
-const SPONSOR_AMOUNT = 250
-const STAKE_AMOUNT = 100
-
 const STREAM_CREATION_KEY = '0xb1abdb742d3924a45b0a54f780f0f21b9d9283b231a0a0b35ce5e455fa5375e7'
 
-// test is outdated, is completely rewritten and will be merged with PR #1629 
-describe.skip('MaintainOperatorValueService', () => {
+async function getTotalUnwithdrawnEarnings(operatorContract: Operator): Promise<bigint> {
+    const { earnings } = await operatorContract.getEarningsFromSponsorships()
+    let unwithdrawnEarnings = BigInt(0)
+    for (const e of earnings) {
+        unwithdrawnEarnings += e.toBigInt()
+    }
+    logger.debug(`Total unwithdrawn earnings: ${formatEther(unwithdrawnEarnings.toString())} DATA (t = ${Date.now()})`)
+    return unwithdrawnEarnings
+}
+
+describe('MaintainOperatorValueService', () => {
     let provider: Provider
-    let operatorWallet: Wallet
-    let operatorContract: Operator
     let token: TestToken
-    let streamId1: string
-    let streamId2: string
-
-    let operatorConfig: OperatorServiceConfig
-
-    const getDiffBetweenApproxAndRealValues = async (): Promise<bigint> => {
-        const { sponsorshipAddresses, approxValues, realValues } = await operatorContract.getApproximatePoolValuesPerSponsorship()
-        let totalDiff = BigInt(0)
-        for (let i = 0; i < sponsorshipAddresses.length; i++) {
-            const diff = realValues[i].toBigInt() - approxValues[i].toBigInt()
-            totalDiff += diff
+    let streamId: string
+
+    const deployNewOperator = async () => {
+        const operatorWallet = await generateWalletWithGasAndTokens(provider, chainConfig)
+        logger.debug('Deploying operator contract')
+        const operatorContract = await deployOperatorContract(chainConfig, operatorWallet, { operatorSharePercent: 10 })
+        logger.debug(`Operator deployed at ${operatorContract.address}`)
+        const operatorConfig = {
+            operatorContractAddress: toEthereumAddress(operatorContract.address),
+            provider,
+            theGraphUrl,
+            signer: operatorWallet,
+            maxSponsorshipsCount: 20,
+            minSponsorshipEarnings: 1 // full tokens
         }
-        return totalDiff
+        return { operatorWallet, operatorContract, operatorConfig }
     }
```

Comment:
Can we use pre-existing `setupOperatorContract` for this? (To avoid code repetition)

Reply:
Couldn't use that one because { operatorSharePercent: 10 } param was needed